pkgs/by-name/mo/monaspace/fonts.toml

```
@@ -0,0 +1,38 @@
+version = "1.301"
+baseUrl = "https://github.com/githubnext/monaspace/releases/download"
+fontFamily = "monaspace"
+
+[[fonts]]
+hash = "sha256-H6J4InGyXabZuslywuzNYqw14zymzF90JKxa7CikOIM="
```

This is the hash that I get when I try to build it.  Please verify.
```suggestion
hash = "sha256-/A5/zipWaaSpTGUBVZOE0EihJY9tu0JAH+6+gyHbDxo="
```